torch/csrc/distributed/rpc/process_group_agent.h

```diff
@@ -264,6 +264,10 @@ class ProcessGroupAgent : public RpcAgent {
   std::mutex metricsMutex_;
   std::vector<std::unique_ptr<AverageMetricsTracker>> metrics_;
   void addGilWaitTime(const std::chrono::microseconds gilWaitTime) override;
+
+  std::atomic<int32_t> clientActiveCalls_{0};
+  std::atomic<int32_t> serverActiveCalls_{0};
+  std::atomic<int32_t> serverActiveAsyncCalls_{0};
```

Comment:
Any reason why these are int32_t and not uint64_t ?

Reply:
I don't think we necessarily expect 2^31 active rpc calls in a given client or server instance at once... :)

(totally agree that int64_t makes sense for sums, but felt overkill for this, since we'll OOM a T6 before we get anywhere near the int32 limit)